tests/components/participants/hooks/useParticipants.test.ts

```diff
@@ -0,0 +1,69 @@
+import { act, renderHook, waitFor } from '@testing-library/react';
+
+import { loadParticipants } from '../../../../src/components/participants/helpers/participantDataFlow';
+import { useParticipants } from '../../../../src/components/participants/hooks/useParticipants';
+
+import { mockParticipants } from './__fixtures__/participants';
+
+jest.mock('../../../../src/components/participants/helpers/participantDataFlow', () => ({
+  loadParticipants: jest.fn(),
+}));
+
+describe('useParticipants', () => {
+  beforeEach(() => {
+    jest.clearAllMocks(); // Clear any previous mocks
+  });
+
+  it('should return LOADING initially', async () => {
```

Comment:
Please remove this test. I think it's just pure luck that you could test inbetween state of the application. I think this test is unrilayable. You still get 100% coverage.